### ISSUE TITLE
Build github actions to run tests, sonar and code validations (pylint and flake8)

### ISSUE THREAD

**Issue by michaelwiles** [didx-xyz/aries-cloudapi-python#8]
Need to ensure that tests and code validations run on each checkin.

**Comment by mrcbond88** [comment]
Will move to review once Mo is done with Feature Auth 7

**Comment by michaelwiles** [comment]
this is actually been done with #9

**Comment by michaelwiles** [comment]
Gonna close this as this is done. Any issues raise another issue.
